### PR TITLE
Lab3

### PR DESCRIPTION
**Pull request by emileyray**

### CODE REVIEW

**Comment by al1ych** [comment]
cool that you made it so that it runs workflow only when the corresponding folder has been changed

could not find any other problems, lgtm

**Review by MikhailMarch — APPROVED** [review]
LGTM
As a recommendation: you can increase timeout for your jobs

**Comment by Dablup** [comment]
Looks good!

Try to add structure(organise files in folders) to your project, it will increase readability of your code

File: .github/workflows/python.yml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+defaults:
+  run:
+    working-directory: ./app_python
+
+jobs:
+  changes:
+    runs-on: ubuntu-latest
+    outputs:
+      backend: ${{ steps.filter.outputs.backend }}
```
Comment (SMore-Napi) [inline .github/workflows/python.yml:13]: Quite ambiguous  names: backend and frontend

**Review by SMore-Napi — COMMENTED** [review]
Looks nice

**Pull request closed without merging**